api/v1alpha1/humiocluster_types.go

```diff
@@ -310,6 +312,35 @@ type HumioNodePoolSpec struct {
 	HumioNodeSpec `json:"spec,omitempty"`
 }
 
+// PodDisruptionBudgetSpec defines the desired pod disruption budget configuration
+type HumioPodDisruptionBudgetSpec struct {
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Format=int-or-string
+	// +kubebuilder:validation:Immutable
+	// MinAvailable is the minimum number of pods that must be available during a disruption.
+	MinAvailable *intstr.IntOrString `json:"minAvailable,omitempty"`
+
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Format=int-or-string
+	// +kubebuilder:validation:Immutable
```

Comment:
What is the idea behind making this `Immutable`? I don't see why we would not allow users to change the configs after initially configuring.

Reply:
Same answer as above